U17 notice

bradleycamacho opened the pull request.

bradleycamacho committed:
u17 notice page

bradleycamacho committed:
Clarify

bradleycamacho committed:
Copy edits

bradleycamacho committed:
Fix activation date

bradleycamacho committed:
Update blobGasUsed info

bradleycamacho committed:
fix formatting

geoknee commented on notices/upgrade-17.mdx:
```diff
@@ -0,0 +1,91 @@
+
+Update your node software as soon as possible after release:
+* Update `op-node` to [NEED VERSION]
+* Update `op-geth` to [NEED VERSION]
```
We will want to call out versions for
* op-reth
* kona node/host/client
* op-challenger.

bradleycamacho replied:
Do we have these yet or will we have them once we're code complete?

geoknee replied:
TODO We also need to release a new op-conductor and place the version here.

geoknee commented on notices/upgrade-17.mdx:
```diff
@@ -0,0 +1,91 @@
+
+* **Cannon is being updated to support Go 1.24**, which allows the OP Stack to stay up to date with upstream changes to go-ethereum (which are included in op-geth)
+* **Minimum Base Fee and Data Availability Footprint Block Limit** which allows the L2 base fee to respond more rapidly to changes in block composition, particularly when this implies an increased demand on the rollup’s capacity to batch blocks to the data availability layer.
+* **Operator Fee Fix** which modifies the existing, optional “operator fee” feature.
```
We should also make a special callout for anyone running operator fee with nonzero scalars. They need to adjust the scalars before activating the fork to avoid overcharging users. Recommend they examine the specs to understand what adjustment to make.

geoknee commented on notices/upgrade-17.mdx:
```diff
@@ -0,0 +1,91 @@
+Upgrade 17 introduces the following changes:
+
+* **Cannon is being updated to support Go 1.24**, which allows the OP Stack to stay up to date with upstream changes to go-ethereum (which are included in op-geth)
+* **Minimum Base Fee and Data Availability Footprint Block Limit** which allows the L2 base fee to respond more rapidly to changes in block composition, particularly when this implies an increased demand on the rollup’s capacity to batch blocks to the data availability layer.
```
I would split these up into two bullets.

Be good to clarify default behavior for chain operators:
* da footprint: this can be disabled by setting a zero scalar on chain, but it will be enabled by default (see the specs) 
* min base fee is disabled by default

Wazabie committed:
Revise Upgrade 17 details in upgrade-17.mdx

Updated activation times and chains for Upgrade 17.

Wazabie committed:
Update upgrade-17.mdx for Upgrade 17 changes

Upgrade 17 introduces support for Go 1.24, configurable minimum base fee, and data availability footprint block limit. It also includes an operator fee fix for more flexible configurations.

Wazabie committed:
Update upgrade-17.mdx

Wazabie committed:
Update upgrade notice for clarity on activations

Clarified language regarding on-chain upgrade requirements.

Wazabie committed:
Update upgrade-17.mdx

sbvegan commented on notices/upgrade-17.mdx:
```diff
@@ -0,0 +1,91 @@
+---
+title: Upgrade 17 - Jovian Hard Fork
```
I think the governance team is still collecting feedback and I don't think a decision has been made yet

sbvegan committed:
Merge branch 'mintlify' into u17-notice

sbvegan committed:
making some updates

pauldowman commented on notices/upgrade-17.mdx:
```diff
@@ -0,0 +1,105 @@
+
+Chain operators must complete the following tasks:
+
+* Update `op-challenger` to [NEED VERSION]
```
[v1.7.0-rc.1](https://github.com/ethereum-optimism/optimism/releases/tag/op-challenger%2Fv1.7.0-rc.1)

pauldowman commented on notices/upgrade-17.mdx:
```diff
@@ -0,0 +1,105 @@
+<Steps>
+  <Step title="Verify the new absolute prestate">
+
+  The absolute prestate is generated with the [op-program/v1.8.0-rc.3](https://github.com/ethereum-optimism/optimism/tree/op-program/v1.8.0-rc.3). You can use this new absolute prestate `0x03799051d2bfe459127d4597f469f535ff1bd2a6e1e2134443167620871c11f3` for the following chains:
```
✅

pauldowman commented on notices/upgrade-17.mdx:
```diff
@@ -0,0 +1,105 @@
+
+  The absolute prestate is generated with the [op-program/v1.8.0-rc.3](https://github.com/ethereum-optimism/optimism/tree/op-program/v1.8.0-rc.3). You can use this new absolute prestate `0x03799051d2bfe459127d4597f469f535ff1bd2a6e1e2134443167620871c11f3` for the following chains:
+
+  *   Mainnet and Sepolia: `OP`, `Base`, `Soneium`, `Ink`, `Unichain`, `Metal`, `Mode`, `Zora`, `Arena-Z`, and `Swell` (Swell Mainnet only).
```
The [prestate review](https://www.notion.so/oplabs/U17-29af153ee1628064b946e505b2054c47?v=1bdf153ee1628012962d000c0011a34a&source=copy_link) was done for these chains:
op-sepolia,ink-sepolia,base-sepolia,unichain-sepolia,soneium-minato-sepolia,op-mainnet,ink-mainnet,base-mainnet,unichain-mainnet,soneium-mainnet

The others are permissioned, and probably aren't running a challenger. (Soneium is also permissioned for now, but permissionless soon).

pauldowman commented on notices/upgrade-17.mdx:
```diff
@@ -0,0 +1,105 @@
+    During the previous step, you also generated the preimage of the absolute prestate, which is the op-program serialized into a binary file. You'll find that new file at `optimism/op-program/bin/prestate-mt64.bin.gz`. Rename that file to have the absolute prestate hash as the filename so it looks like `PRESTATEHASH.bin.gz`.
+
+    Upload that file to where you're storing your other absolute preimage files. This should be the location where you're pointing your `--cannon-prestates-url` at. The `op-challenger` will grab this file and use it when it needs to challenge games.
+  </Step>
```
The prestate section looks good.

sbvegan committed:
clean up

sbvegan committed:
addressing Paul's review

geoknee commented on notices/upgrade-17.mdx:
```diff
@@ -0,0 +1,143 @@
+* **Cannon Go 1.24 support:** upgrading the on-chain fault proof virtual machine implementation to support Go 1.24.
+* **Configurable Minimum Base Fee:** allows chain operators to specify a minimum base fee to shorten the length of priority fee auctions (disabled by default).
+* **Data Availability Footprint Block Limit:** adds an in-protocol limit to estimated DA usage of transactions to prevent DA spam and priority fee auctions. (enabled by default; can be disabled via zero scalar on-chain).
+* **Operator Fee Fix:** enables more flexible fee configurations for rollup operators and lays groundwork for future custom gas token support. If you are running operator fee with nonzero scalars, make sure to adjust them before activating the fork to avoid overcharging users.
```
Perhaps this should be a callout since it is a specific call to action?

Wazabie committed:
Update upgrade-17.mdx

Wazabie committed:
Fix formatting of Operator Fee Fix section

sbvegan committed:
updating links
